### ISSUE TITLE
Screen Reader: Sign in: The button in the picker view is announced without a button role

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77546]
## Action Performed:
Prerequisites:
- the user is not signed in

1. Using iOS VoiceOver, open the Expensify app
2. Swipe to the 'Language' dropdown and double-tap to activate
3. Swipe to the 'Done' button
4. Note how the control is announced

## Expected Result:
The control should be announced with a button role.

## Actual Result:
The button in the picker view is announced without a button role.

## Area issue was found in:
Sign in

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Apply accessibilityRole="button" to the interactive component.
https://reactnative.dev/docs/button

## Environment:
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

<img width="828" height="1792" alt="Image" src="https://github.com/user-attachments/assets/09ba7bc7-6f30-4861-ba0a-c904d2fcf3ba" />

https://github.com/user-attachments/assets/c35f727c-0ce4-47d5-916e-88743468c6b9

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005377
**Issue reported by:** NA
**Slack conversation:** NA
